Implementations/Uavcan.rs/index.md

```diff
@@ -0,0 +1,7 @@
+---
+weight: 3
+---
+
+# Uavcan.rs
+
+Uavcan.rs aims to be a full feature uavcan implementation in [Rust](www.rust-lang.org). You can contribute or follow the development at [Github](https://github.com/UAVCAN/uavcan.rs), download the latest version with cargo or from [crates.io](https://crates.io/crates/uavcan), or read the documentation at [docs.rs](https://docs.rs/uavcan/).
```

Comment:
"full feature" should perhaps be "full featured"?

`[Rust](www.rust-lang.org)` - is this a proper URL? The scheme is not specified. Shouldn't it be `[Rust](http://www.rust-lang.org)`?